src/foam/core/AbstractArrayPropertyInfo.java

```diff
@@ -63,10 +63,17 @@ public Object fromXML(X x, XMLStreamReader reader) {
   public void toXML (FObject obj, Document doc, Element objElement) {
     if ( this.f(obj) == null ) return;
 
+    try {
```

Comment:
Add a comment to explain what this is for.

Reply:
@kgrgreer When select EmailTemplateDAO It throws error caused by failing to cast from FObject to Object[], even the FObject parameter is not NULL, or invalid. I guess inside EmailTemplateDAO It has some a bunch of special characters not to cast. So I added above.